src/app/medInria/areas/browser/medBrowserArea.cpp

```diff
@@ -2,7 +2,7 @@
 
  medInria
 
- Copyright (c) INRIA 2013 - 2018. All rights reserved.
+ Copyright (c) INRIA 2013 - 2020. All rights reserved.
```

Comment:
it reminds me that it could be interesting to have a final decision for this topic https://github.com/medInria/medInria-public/issues/543 :)